SRC/about-me/about-me.css

```diff
@@ -0,0 +1,81 @@
+.mainContainer{
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+}
+.backg-Video{
+position: fixed;
+top: 0;
+right: 0;
+bottom: 0;
+left: 0;
+overflow: hidden;
+z-index: -100;
+width: 100%;
+}
+
+
+video {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
```

Comment:
change all this to this: 
`position: fixed;`
`right: 0;`
`bottom: 0;`
`height: 100%;`
`min-height: 100%;`